roles/kibana/proxy/tasks/main.yml

```diff
@@ -5,13 +5,27 @@
     dest: '{{ kibana_httpd_conf }}'
   notify: restart httpd
 
+- name: check if kibana_proxy_user is contained in kibana_proxy_htpasswd
```

Comment:
'we want to run the "Create kibana htpasswd file" task if the password change' >>> On that I beg to differ. If we have a htpasswd with the user who might potentially change his password manually, I would not change it to the one set in playbook by potentially another user rerunning playbook for definitely other reason then changing password. IMHO once htpasswd file is there with required user, it should not be changed.